barberia/src/App/pages/signin.css

```diff
@@ -0,0 +1,39 @@
+input[type="email"],
```

Comment:
let's change this to a class for the two

Reply:
fixed